api/src/routers/meals.js

```diff
@@ -162,4 +232,24 @@ mealsRouter.delete(
   })
 );
 
+mealsRouter.get(
+  "/:meal_id/reviews",
+  asyncHandler(async (req, res) => {
+    const mealId = req.params.meal_id;
+    const meal = await knex("Meal").where("Meal.id", mealId).first();
+    if (!meal) return res.status(404).json({ message: "Meal not found" });
+
+    const result = await knex("Review")
+      .join("Meal", "Review.meal_id", "=", "Meal.id")
+      .where("Meal.id", mealId)
+      .select("Review.*");
+
+    if (result.length === 0)
+      return res
+        .status(404)
+        .json({ message: "No reviews found for this meal" });
+    res.status(200).json(result);
```

Comment:
I suggest to be a bit careful with lines like this, because it can easily end up in the wrong place, i.e. inside/outside the body of the if. Usually it's better to use curly braces to wrap the body (of the if), even though either way is correct in terms of syntax.